### PR TITLE
Depend on pangeo chart

### PR DESCRIPTION
**Pull request by jhamman**

### CODE REVIEW

**Comment by jhamman** [comment]
Made possible by https://github.com/yuvipanda/hubploy/commit/7dab0db22f7c79aef01c4c73e798764318817a70

File: pangeo-deploy/values.yaml [hunk 1]
```diff
@@ -7,34 +7,34 @@
...
+        tag: 2019.02.01
+      cpu:
+        limit: 2
+        guarantee: 1
+      memory:
+        limit: 4G
+        guarantee: 2G
+      defaultUrl: "/lab"
+      cmd: jupyter-labhub
+      serviceAccountName: daskkubernetes
```
Comment (jhamman) [inline pangeo-deploy/values.yaml:33]: I think we can remove basically all of this.

Comment (yuvipanda) [reply pangeo-deploy/values.yaml:33]: Because it's present in the pangeo chart?

**Comment by yuvipanda** [comment]
You need to set images.image_config_path in hubploy.yaml as well. LMK how this goes!

**Comment by rabernat** [comment]
Awesome Joe! Thanks for this!

**Pull request merged**